hudi-cli/src/main/java/org/apache/hudi/cli/commands/RepairsCommand.java

```diff
@@ -263,4 +264,30 @@ public String migratePartitionMeta(
         HoodieTableHeaderFields.HEADER_ACTION
     }, rows);
   }
+
+  @CliCommand(value = "repair deprecated partition",
+      help = "Repair deprecated partition (\"default\"). Re-writes data from the deprecated partition into " + PartitionPathEncodeUtils.DEFAULT_PARTITION_PATH)
```

Comment:
Overall looks good. But if we consider more generally, the functionality can essentially be used to rename partitions. So how about making it more like `mv <old_partition> <new_partition>` command?

Reply:
HUDI-4648 for followup. Just in case if we want to land this first because some users might hit this issue.

Reply:
sg. will prioritize landing this and then adding more enchancements.